ui/imports/shared/controls/chat/LinkPreviewCard.qml

```diff
@@ -6,26 +6,25 @@ import StatusQ.Core 0.1
 import StatusQ.Core.Theme 0.1
 import StatusQ.Components 0.1
 
+import shared.controls 1.0
 import shared.status 1.0
 import utils 1.0
 
+import "./private"
+
 CalloutCard {
     id: root
     
-    /// The title of the callout card
-    required property string title
-    required property string description
-    required property string footer
+    //Using data objects defined in the private files to avoid the following bug
+    //https://bugreports.qt.io/browse/QTBUG-89180
+    readonly property LinkData linkData: LinkData { }
```

Comment:
It would, but trying to write this property from the outside results in property not defined error. I guess it's a similar issue with the inline components.
But nonetheless, now it makes sense to have them in separate files because they're reused by `LinksPreviewMiniCard`